### PR TITLE
Feature/feedback based improvements

### PR DESCRIPTION
**Pull request by lagartoverde**
Solved a bug where the classifications filter was not really working and redid the code filtering so now you can select specific sign codes in a multiselect

### CODE REVIEW

**Comment by lagartoverde** [comment]
We may need to discuss how to redo the filters as a whole, but this is a start

File: addon/utils/fetchData.js [hunk 1]
```diff
@@ -11,7 +11,15 @@ function buildFilters({ zonality, type, code, category }) {
     filters.push(`FILTER(?signType = <${type}>)`);
   }
   if (code) {
```
Comment (nvdk) [inline addon/utils/fetchData.js]: if we have multiple now, best to rename to codes instead of code

File: addon/services/roadsign-registry.js [hunk 2]
```diff
@@ -56,6 +56,28 @@ export default class RoadsignRegistryService extends Service {
     }
   }
 
+  @task
+  *searchCode(codeString) {
+    const query = `
```
Comment (nvdk) [inline addon/services/roadsign-registry.js:64]: this no longer debounces, might want to make it a restartable task with a debounce?

File: addon/components/roadsigns-modal.js [hunk 3]
```diff
@@ -119,7 +121,9 @@ export default class RoadsignRegulationCard extends Component {
           ? this.zonalitySelected.value
           : undefined,
         type: this.typeSelected ? this.typeSelected.value : undefined,
-        code: this.codeFilter,
+        code: this.codeFilter
```
Comment (nvdk) [inline addon/components/roadsigns-modal.js]: should probably be renamed to codes now that it's an array

**Review by abeforgit — APPROVED** [review]

**Pull request merged**